crates/matrix-sdk-crypto/src/identities/user.rs

```diff
@@ -1266,18 +1274,103 @@ pub(crate) mod testing {
         OtherUserIdentityData::new(master_key.try_into().unwrap(), self_signing.try_into().unwrap())
             .unwrap()
     }
+
+    /// When we want to test identities that are verified, we need to simulate
+    /// the verification process. This function supports that by simulating
+    /// what happens when a successful verification dance happens and
+    /// providing the /keys/query response we would get when that happened.
+    ///
+    /// signature_upload_request will be the result of calling
+    /// [`super::OtherUserIdentity::verify`].
+    ///
+    /// msk_json should be JSON like: {
```

Comment:
Can we convert this into code and put this into a code fence, that way this will get rendered a bit nicer in various code editors.

Something like:

```
let msk_json = json!({
   ...
```

Reply:
Done in 6ad623f795